paper-fab-speed-dial.html

```diff
@@ -103,6 +111,12 @@
 
 			this.opened = false;
 		},
+		
+		// Protected methods
+		_computeIcon: function(icon, src) {
+			// If we don't have an icon and we don't have a src then we default to the 'add' icon
+			return ((icon === undefined || icon === '') && (src === undefined || src === '')) ? 'add' : undefined;
```

Comment:
- Simplify to `!icon && !src`
- I'm pretty sure you want to return the icon if it's set, e.g. more something like:
```
_computeIcon: function(icon, src) {
  // <paper-fab> requires to send an icon if the src is defined
  if (src) {
    return undefined;
  }
  // Default to add if neither src nor icon are set
  return icon || 'add';
}
```